### DISCUSSION TITLE
Unable to boot Banana Pi M5 Pre-Build image file

### DISCUSSION THREAD

**Discussion by hkkok** [johang/sd-card-images#22] [General]
Hi! Recently i've tried to download boot-banana_pi_m5.bin.gz and debian-bookworm-arm64-iu4wie.bin.gz for my BananaPi M5 board. The system stop until /scripts/local-block and stopped. The message as shown below:

Begin: Waiting for root file system ... [    5.489159] usb 1-1: new high-speed USB device number 2 using xhci-hcd
[    5.639881] usb 1-1: New USB device found, idVendor=2109, idProduct=2817, bcdDevice=90.23
[    5.644505] usb 1-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[    5.650571] usb 1-1: Product: USB2.0 Hub
[    5.655563] usb 1-1: Manufacturer: VIA Labs, Inc.
[    5.700360] hub 1-1:1.0: USB hub found
[    5.701667] hub 1-1:1.0: 4 ports detected
[    5.725410] mmc0: error -84 whilst initialising SD card
[    5.794748] usb 2-1: new SuperSpeed USB device number 2 using xhci-hcd
[    5.887625] usb 2-1: New USB device found, idVendor=2109, idProduct=0817, bcdDevice=90.23
[    5.892355] usb 2-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[    5.898349] usb 2-1: Product: USB3.0 Hub
[    5.903321] usb 2-1: Manufacturer: VIA Labs, Inc.
[    5.939917] hub 2-1:1.0: USB hub found
[    5.941249] hub 2-1:1.0: 4 ports detected
Begin: Running /scripts/local-block ... done.
[    6.813397] mmc0: error -84 whilst initialising SD card
Begin: Running /scripts/local-block ... done.
[    7.901371] mmc0: error -84 whilst initialising SD card
Begin: Running /scripts/local-block ... done.
[    8.989376] mmc0: error -84 whilst initialising SD card
Begin: Running /scripts/local-block ... done.
[   10.077403] mmc0: error -84 whilst initialising SD card
Begin: Running /scripts/local-block ... done.
[   11.165384] mmc0: error -84 whilst initialising SD card
Begin: Running /scripts/local-block ... done.
[   12.253366] mmc0: error -84 whilst initialising SD card
Begin: Running /scripts/local-block ... done.

Can I know how is there any solution? Thanks

**Comment by fgalfa** [comment]
I faced the same problem with banana pi m5.
Boot stops after trying to mount root partition for issues while reading SD Card.

I tried with all debian/ubuntu versions.

[    3.716124] Run /init as init process
Loading, please wait...
Starting systemd-udevd version 252-2
[    4.193386] input: gpio-keys as /devices/platform/gpio-keys/input/input0
[    4.244667] pps_core: LinuxPPS API ver. 1 registered
[    4.244764] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    4.254606] meson-gx-mmc ffe05000.sd: Got CD GPIO
[    4.259399] meson-gx-mmc ffe07000.mmc: allocated mmc-pwrseq
[    4.284906] meson-vrtc ff8000a8.rtc: registered as rtc0
[    4.285923] meson-vrtc ff8000a8.rtc: setting system clock to 1970-01-01T00:00:04 UTC (4)
[    4.328323] PTP clock support registered
[    4.331666] dwc3-meson-g12a ffe09000.usb: USB2 ports: 2
[    4.332783] dwc3-meson-g12a ffe09000.usb: USB3 ports: 1
[    4.367592] dwc3-meson-g12a ffe09000.usb: switching to Device Mode
[    4.388979] mmc0: new high speed SDHC card at address aaaa
[    4.407205] mmcblk0: mmc0:aaaa SC16G 14.8 GiB 
[    4.451793] mmc1: new HS200 MMC card at address 0001
[    4.458476] meson8b-dwmac ff3f0000.ethernet: IRQ eth_wake_irq not found
[    4.460565] meson8b-dwmac ff3f0000.ethernet: IRQ eth_lpi not found
[    4.467555] mmcblk1: mmc1:0001 AJTD4R 14.6 GiB 
[    4.470343] usbcore: registered new interface driver usbfs
[    4.473342] meson8b-dwmac ff3f0000.ethernet: PTP uses main clock
[    4.476152] usbcore: registered new interface driver hub
[    4.490254] usbcore: registered new device driver usb
[    4.490842] mmcblk1boot0: mmc1:0001 AJTD4R 4.00 MiB 
[    4.509462] mmcblk1boot1: mmc1:0001 AJTD4R 4.00 MiB 
[    4.510460] meson8b-dwmac ff3f0000.ethernet: User ID: 0x11, Synopsys ID: 0x37
[    4.515699] mmcblk1rpmb: mmc1:0001 AJTD4R 4.00 MiB, chardev (242:0)
[    4.516053] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    4.517483] meson8b-dwmac ff3f0000.ethernet:         DWMAC1000
[    4.517494] meson8b-dwmac ff3f0000.ethernet: DMA HW capability register supported
[    4.517499] meson8b-dwmac ff3f0000.ethernet: RX Checksum Offload Engine supported
[    4.517503] meson8b-dwmac ff3f0000.ethernet: COE Type 2
[    4.517508] meson8b-dwmac ff3f0000.ethernet: TX Checksum insertion supported
[    4.517512] meson8b-dwmac ff3f0000.ethernet: Wake-Up On Lan supported
[    4.517604] meson8b-dwmac ff3f0000.ethernet: Normal descriptors
[    4.517610] meson8b-dwmac ff3f0000.ethernet: Ring mode enabled
[    4.517615] meson8b-dwmac ff3f0000.ethernet: Enable RX Mitigation via HW Watchdog Timer
[    4.524509] dwc2 ff400000.usb: supply vusb_d not found, using dummy regulator
[    4.530487] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    4.534938] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    4.535940] dwc2 ff400000.usb: supply vusb_a not found, using dummy regulator
[    4.540317] meson8b-dwmac ff3f0000.ethernet end0: renamed from eth0
[    4.543108] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    4.550861] dwc2 ff400000.usb: EPs: 7, dedicated fifos, 712 entries in SPRAM
[    4.560341] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    4.597788] xhci-hcd xhci-hcd.1.auto: xHCI Host Controller
[    4.602767] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    4.611133] xhci-hcd xhci-hcd.1.auto: new USB bus registered, assigned bus number 1
[    4.618202]  mmcblk0: unable to read partition table
[    4.624603] xhci-hcd xhci-hcd.1.auto: hcc params 0x0228fe6c hci version 0x110 quirks 0x0000002000010010
[    4.695804] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[    4.697839] xhci-hcd xhci-hcd.1.auto: irq 20, io mem 0xff500000
[    4.702754] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    4.706906] xhci-hcd xhci-hcd.1.auto: xHCI Host Controller
[    4.715080] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    4.720556] xhci-hcd xhci-hcd.1.auto: new USB bus registered, assigned bus number 2
[    4.744670] xhci-hcd xhci-hcd.1.auto: Host supports USB 3.0 SuperSpeed
[    4.746943] usb usb1: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 6.00
[    4.756134] usb usb1: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    4.762142] usb usb1: Product: xHCI Host Controller
[    4.766981] usb usb1: Manufacturer: Linux 6.0.0-2-arm64 xhci-hcd
[    4.772931] usb usb1: SerialNumber: xhci-hcd.1.auto
[    4.778419] hub 1-0:1.0: USB hub found
[    4.781555] hub 1-0:1.0: 2 ports detected
[    4.786033] usb usb2: We don't know the algorithms for LPM for this host, disabling LPM.
[    4.794951] usb usb2: New USB device found, idVendor=1d6b, idProduct=0003, bcdDevice= 6.00
[    4.803032] usb usb2: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    4.808897] usb usb2: Product: xHCI Host Controller
[    4.813694] usb usb2: Manufacturer: Linux 6.0.0-2-arm64 xhci-hcd
[    4.819644] usb usb2: SerialNumber: xhci-hcd.1.auto
[    4.825141] hub 2-0:1.0: USB hub found
[    4.828203] hub 2-0:1.0: 1 port detected
Begin: Loading essential drivers ... done.
Begin: Running /scripts/init-premount ... done.
Begin: Mounting root file system ... Begin: Running /scripts/local-top ... done.
Begin: Running /scripts/local-premount ... done.
[    4.937528] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[    4.944977] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    4.951538] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    5.049292] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[    5.056854] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    5.063349] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Waiting for root file system ... [    5.234533] usb 1-1: new high-speed USB device number 2 using xhci-hcd
[    5.385170] usb 1-1: New USB device found, idVendor=2109, idProduct=2817, bcdDevice=90.23
[    5.390221] usb 1-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[    5.396055] usb 1-1: Product: USB2.0 Hub             
[    5.401042] usb 1-1: Manufacturer: VIA Labs, Inc.         
[    5.451317] hub 1-1:1.0: USB hub found
[    5.452763] hub 1-1:1.0: 4 ports detected
[    5.514922] usb 2-1: new SuperSpeed USB device number 2 using xhci-hcd
[    5.644993] usb 2-1: New USB device found, idVendor=2109, idProduct=0817, bcdDevice=90.23
[    5.650010] usb 2-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[    5.655910] usb 2-1: Product: USB3.0 Hub             
[    5.660869] usb 2-1: Manufacturer: VIA Labs, Inc.         
[    5.691285] hub 2-1:1.0: USB hub found
[    5.692826] hub 2-1:1.0: 4 ports detected
Begin: Running /scripts/local-block ... done.
[    6.165694] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[    6.173253] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    6.268899] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[    7.371645] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    7.464202] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
Begin: Running /scripts/local-block ... done.
[    9.757612] blk_print_req_error: 11 callbacks suppressed
[    9.757630] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[    9.772010] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    9.777197] buffer_io_error: 2 callbacks suppressed
[    9.777202] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[    9.876838] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[    9.884969] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[    9.890709] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   10.993945] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   11.001073] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   11.007847] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   11.105241] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   11.112722] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   11.119307] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   12.226032] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   12.233097] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   12.240204] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   12.338543] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   13.440479] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   13.535112] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   14.639862] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   14.735579] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   15.071832] platform adc_keys: deferred probe pending
[   15.073160] platform regulator-vddcpu: deferred probe pending
Begin: Running /scripts/local-block ... done.
[   15.836934] blk_print_req_error: 10 callbacks suppressed
[   15.836950] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   15.852739] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   15.858117] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   15.952932] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   15.963796] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   15.969226] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   17.073954] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   17.081726] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   17.088122] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   17.184734] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   17.192405] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   17.198899] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   18.302598] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   18.309650] I/O error, dev mmcblk0, sector 0 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   18.316757] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   18.414489] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   19.572165] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   19.667100] Buffer I/O error on dev mmcblk0, logical block 0, async page read
Begin: Running /scripts/local-block ... done.
[   20.770692] Buffer I/O error on dev mmcblk0, logical block 0, async page read
[   20.860919] mmc0: tried to HW reset card, got error -84
[   20.863662] blk_print_req_error: 9 callbacks suppressed
[   20.863670] I/O error, dev mmcblk0, sector 31116272 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   20.881309] I/O error, dev mmcblk0, sector 31116272 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   20.887439] Buffer I/O error on dev mmcblk0, logical block 3889534, async page read
Begin: Running /scripts/local-block ... done.
[   21.936336] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   21.944466] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   21.951866] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   21.992215] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   22.000320] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   22.007695] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   23.057892] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   23.067592] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   23.073743] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   23.115041] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   23.125536] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   23.131676] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   24.183461] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   24.222935] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   25.272254] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   25.312120] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   26.359077] blk_print_req_error: 8 callbacks suppressed
[   26.359090] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   26.374582] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   26.380743] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   26.420760] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   26.428729] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   26.436266] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   27.486369] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   27.496050] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   27.502236] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   27.542744] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   27.551967] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   27.558254] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   28.607546] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   28.617423] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   28.623648] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   28.664963] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   29.712656] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   29.751391] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   30.799503] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   30.838064] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   31.884374] blk_print_req_error: 10 callbacks suppressed
[   31.884388] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   31.900668] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   31.906784] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   31.946897] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   31.954769] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   31.962312] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   33.012112] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   33.021845] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   33.027984] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   33.068347] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   33.076070] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   33.083776] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   34.134772] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[   34.144366] I/O error, dev mmcblk0, sector 31116160 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[   34.150458] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   34.192862] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Begin: Running /scripts/local-block ... done.
[   35.239870] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   35.279486] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
done.
[   35.323883] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
[   35.363860] Buffer I/O error on dev mmcblk0, logical block 3889520, async page read
Gave up waiting for root file system device.  Common problems:
 - Boot args (cat /proc/cmdline)
   - Check rootdelay= (did the system wait long enough?)
 - Missing modules (cat /proc/modules; ls /dev)
ALERT!  PARTUUID=8a11d710-02 does not exist.  Dropping to a shell!
(initramfs) [   39.522523] random: crng init done
(initramfs) [  161.374710] mmc0: card aaaa removed
[  917.288088] mmc0: new high speed SDHC card at address aaaa
[  917.290247] mmcblk0: mmc0:aaaa SC16G 14.8 GiB 
[  917.297473]  mmcblk0: p1 p2
[  917.361633] blk_print_req_error: 10 callbacks suppressed
[  917.361650] I/O error, dev mmcblk0, sector 8 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2
[  917.376815] I/O error, dev mmcblk0, sector 8 op 0x0:(READ) flags 0x0 phys_seg 1 prio class 2
[  917.382093] Buffer I/O error on dev mmcblk0, logical block 1, async page read
[  917.455824] I/O error, dev mmcblk0, sector 65544 op 0x0:(READ) flags 0x80700 phys_seg 1 prio class 2

**Comment by johang** [comment]
So bootloader works, kernel startup works, probing the SD card works (mmc0:aaaa SC16G 14.8 GiB), but reading from it fails. I suspect some reconfiguration in the Debian kernel. Possibly something in the power management or clocking.

**Comment by johang** [comment]
Try the Debian Sid image.

Possibly fixed by this: https://patchwork.kernel.org/project/linux-amlogic/patch/20230127142221.3718184-2-christianshewitt@gmail.com/

**Reply by nullsauce** [reply]
Hi @johang. Thanks for all the images. Super handy! 

I'm encountering the same issue on a Banana PI M5. Tried Debian Sid (`2023-08-07`) but the issues is still there. Any advice ?
